docs/code/MlNetCookBook.md

```diff
@@ -303,6 +303,52 @@ private class InspectedRow
 }
 ```
 
+You can also use the dynamic API to create the equivalent of the previous pipeline. 
+```csharp
+// Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+// as a catalog of available operations and as the source of randomness.
+var mlContext = new MLContext();
+
+// Create the reader: define the data columns and where to find them in the text file.
+var reader = new TextLoader(mlContext, new TextLoader.Arguments
```

Comment:
>TextLoader [](start = 17, length = 10)

mlContext.Data.TextReader

Reply:
updated them all.

---
In reply to: [231017639](https://github.com/dotnet/machinelearning/pull/1538#discussion_r231017639) [](ancestors = 231017639)